locales/en-US/app.properties

```diff
@@ -26,6 +26,7 @@ documents=Documents
 documents-device=Documents on this Device
 documents-dropbox=Documents on your Dropbox
 done=Done
+dropbox-connect=Connect to Dropbox
```

Comment:
I don't think we should add another string right now. Can we use `use-dropbox` instead? (And we can change that string later).